Application/EdFi.Ods.AdminApi/Features/RequestLoggingMiddleware.cs

```diff
@@ -14,24 +15,25 @@ namespace EdFi.Ods.AdminApi.Features;
 public class RequestLoggingMiddleware
 {
     private readonly RequestDelegate _next;
+    private static readonly ILog _logger = LogManager.GetLogger(typeof(RequestLoggingMiddleware));
 
     public RequestLoggingMiddleware(RequestDelegate next)
     {
         _next = next ?? throw new ArgumentNullException(nameof(next));
     }
 
-    public async Task Invoke(HttpContext context, ILogger<RequestLoggingMiddleware> logger)
+    public async Task Invoke(HttpContext context)

```

Comment:
This did not use dependency injection, but that's OK in this case, since we're not unit testing logging.